bot/cogs/coding/coding.py

```diff
@@ -68,8 +70,10 @@ async def update_languages(self) -> None:
         "rust": documentation.rust_doc,
     }
 
-    @commands.command(
-        help="""
+    @commands.command()
+    async def run(self, ctx: Context, language: str, *, code: str = "") -> t.Union[None, str]:
+        """Execute code in a given programming language.
+
```

Comment:
That's not what I saw, and that is the regular way to do it : first line is the short description, but you make it the second line

Reply:
While yes, technically we're starting on the 2nd line it greatly improves on readability it's worth it.
Also, discord uses these docstrings as function description and it ignores this first blank line anyway so we don't need to avoid it and considering the readability advantages, it seems quite reasonable to follow this.

Reply:
I prefer doing things the `Rapptz` way

Reply:
I get that and it's not a bad way but I'd still go for readability in this case and this way is better in that aspect so I still insist on having this followed as suggested. 

Also, since this has the potential to be a more common issue I'll include this docstring formatting in `CONTRIBUTING.md` to keep it clear as to which format should be used

Reply:
Done, you can find the docstring syntax guidelines documented in [`CONTRIBUTING.md/Docstring Formatting Directive`](https://github.com/The-Codin-Hole/HotWired-Bot/blob/master/CONTRIBUTING.md#docstring-formatting-directive)

Reply:
Really, Rapptz does it that way, even my linter says that it's better done that way. What do you want more ?

Reply:
As I've said multiple times now, we simply don't follow this method and we have a very specific docstring convention.
What I want more is for you to respect it too, the fact that Rapptz does it in this way is possible but not relevant in this case, we're aware that our docstring structure is a bit unique but that's not necessarily a bad thing and as I've already said, it provides better readability and it's simply the convention for this repository.